manifests/o/OctopusDeploy/Tentacle/6.1.1318/OctopusDeploy.Tentacle.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OctopusDeploy.Tentacle
+PackageVersion: 6.1.1318
```

Version is still 1.0.0.0
```suggestion
PackageVersion: 1.0.0.0
```